### PR TITLE
Fixed breaking changes from the new SDK, removed dependency on helper code.

### PR DESCRIPTION
**Pull request by sanjeev3**
…tor helper.

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (yangyuan) [inline .vs/VSWorkspaceState.json]: Please remove all .vs folders.

(I will add gitignore as well)

Comment (sanjeev3) [reply .vs/VSWorkspaceState.json]: @yangyuan, Done. Thanks! Can you please check?

Comment (yangyuan) [reply .vs/VSWorkspaceState.json]: On .vs is still there... in the root folder :) 
/.vs/

**Comment by sanjeev3** [comment]
@yangyuan, I removed the .vs from the root folder. Try now. :)

**Comment by yangyuan** [comment]
all good, merged.

**Pull request merged**